Source/Core/Common/JitRegister.h

```diff
@@ -10,23 +11,22 @@ namespace JitRegister
 {
 void Init(const std::string& perf_dir);
 void Shutdown();
-void RegisterV(const void* base_address, u32 code_size, const char* format, va_list args);
+void RegisterV(const void* base_address, u32 code_size, fmt::string_view format,
+               fmt::format_args args);
 bool IsEnabled();
 
-inline void Register(const void* base_address, u32 code_size, const char* format, ...)
+template <typename... Args>
+inline void Register(const void* base_address, u32 code_size, fmt::string_view format,
+                     const Args&... args)
 {
-  va_list args;
-  va_start(args, format);
-  RegisterV(base_address, code_size, format, args);
-  va_end(args);
+  RegisterV(base_address, code_size, format, fmt::make_format_args(args...));
 }
```

Comment:
With slightly different syntax, it'd be possible to use `fmt`'s compile-time format checking, which only applies on C++20 but the MSVC builder targets C++20 so we'd still benefit from it:

```C++
template <typename... Args>
inline void Register(const void* base_address, u32 code_size, fmt::format_string<Args...> format,
                     Args&&... args)
{
  Register(base_address, code_size, fmt::format(format, std::forward<Args>(args)...));
}
```

The same would apply to the other `Register` function below.